App/HejDTO/Tests/HejDTOTests/HejDTOTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+@testable import HejDTO
+
+final class HejDTOTests: XCTestCase {
+    func testExample() throws {
```

Comment:
It would be cool to have an actual test for that. They have documentation with example requests/responses. You can use it to check if the JSON they provide will actually serialize.